### PR TITLE
use FunctionLike Interface

### PR DESCRIPTION
**Pull request by jens1o**
because why not?

### CODE REVIEW

**Comment by jens1o** [comment]
Why does that decrease coverage, while actually cleaning up and using all features of the PhpParser

**Comment by MadHed** [comment]
@jens1o Because you removed 2 lines, meaning the uncovered lines have a relatively bigger impact. ;)

**Comment by jens1o** [comment]
Ah, that's making sense. Thanks :)

File: src/ParserHelpers.php [hunk 1]
```diff
@@ -40,10 +40,7 @@ function getFunctionLikeDeclarationFromParameter(Node\Parameter $node)
 
 function isFunctionLike(Node $node)
```
Comment (felixfbecker) [inline src/ParserHelpers.php]: we can get rid of this function then

Comment (jens1o) [reply src/ParserHelpers.php]: Okay, I though we're leaving it as a shortcut. Will do this tomorrow.

**Pull request merged**